src/core/control/pagetype/PageTypeMenu.cpp

```diff
@@ -186,13 +193,17 @@ void PageTypeMenu::addApplyBackgroundButton(PageTypeApplyListener* pageTypeApply
     menuY++;
 
     if (!onlyAllMenu) {
-        GtkWidget* menuEntryApply = createApplyMenuItem(_("Apply to current page"));
+        menuEntryApply = createApplyMenuItem(_("Apply to current page"));
```

Comment:
Why was this converted into a member field?

Reply:
Note that `addApplyBackgroundButton` can be called in multiple places, so this could potentially be overridden.